docs/basics/beginners-contributing-guide.md

```diff
@@ -0,0 +1,117 @@
+# Beginners Contributing Guide
+
```

Comment:
Mention the "Contribution guidelines" that are in CONTRIBUTING.md
Also CONTRIBUTING.md should also mention this article

Reply:
@py-crash haha good call ill added it